### PR TITLE
DOC: Example for signal.cmplx_sort

### PR DESCRIPTION
**Pull request by ziejcow**
Reference to: #7168

### CODE REVIEW

File: scipy/signal/signaltools.py [hunk 1]
```diff
@@ -1662,6 +1662,16 @@ def cmplx_sort(p):
     indx : ndarray
         Array of indices needed to sort the input `p`.
 
+    Examples
+    --------
+    >>> from scipy import signal
+    >>> vals = [1, 4, 1+1.j, 3]
+    >>> p_sorted, indx = signal.cmplx_sort(vals)
+    >>> p_sorted
+    array([1.0+0.j, 1.0+1.j, 3.+0.j, 4.+0.j])
```
Comment (WarrenWeckesser) [inline scipy/signal/signaltools.py]: Is this the actual output that you get when you display `p_sorted`?   With numpy 1.14.1, I get

    array([1.+0.j, 1.+1.j, 3.+0.j, 4.+0.j])

Note that the real parts of the first two values are displayed as `1.`, not `1.0`.

With numpy 1.13.1, I get

    array([ 1.+0.j,  1.+1.j,  3.+0.j,  4.+0.j])

Comment (ziejcow) [reply scipy/signal/signaltools.py]: I just checked it and indeed that's not the case.
The output with version 1.8.2 is: 
`array([ 1.+0.j,  1.+1.j,  3.+0.j,  4.+0.j])`
Just as you pointed out. I just pushed desirable changes into this branch.

**Comment by pv** [comment]
Note that the documentation should agree with the latest numpy release. 
Not that whitespace matters that much, but there's an automatic check 
that needs to pass, and it may not be smart enough to ignore the 
whitespace here.

**Comment by ziejcow** [comment]
After updating numpy I fixed whitespaces to match current version.
AFAIK all checks passed even in previous version of this PR and hopefully they will pass in this as well.

**Comment by ilayn** [comment]
Thanks @ziejcow

**Pull request merged**